core/javacc.gradle

```diff
@@ -14,7 +14,7 @@ task downloadJavacc {
 
 task installJavacc(type:Copy,dependsOn:downloadJavacc) {
     ext.archive = downloadJavacc.dest
-    ext.javaccHome = file("$buildDir/javacc/")
+    ext.javaccHome = file("$buildDir/javacc/javacc-5.0")
```

Comment:
at the beginning I got problems during the build but currently is working fine. I will remove this.